Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -137,19 +141,37 @@ To calculate the carbon intensity we need to decide two things.
 
 ## Taxonomy of Green Software
 
-The taxonomy of Green Software is broadly split into these categories.
+Green Software broadly addresses emissions in these categories.
 
-- **Carbon Aware Application**: Applications that change behaviour to use the cleanest energy possible, for instance a laptop that charged only when there is lots of renewable power currently being generated.
-- **Energy Efficient Applications:** Applications that use less energy to perform the same function.
+***Operational Emissions:***
+- **Carbon Aware Application**: Applications that change behaviour to use the cleanest energy possible, for instance a laptop that charged only when there is lots of renewable power currently available.
+- **Energy Efficient Applications:** Applications that use less energy, and thus causes less emissions, to perform the same function.
+
+***Embodied Emissions***
 - **Hardware Efficient Applications:** Applications that use the least physical resources possible to perform their function.
 
 ## Characteristics for calculating total carbon emissions, C
 
 ### (1) `C` should be sensitive to carbon awareness, energy efficiency or hardware efficiency improvements to the application
 
-The challenges with existing methods of calculation is that they are only sensitive to some of the above methods of making an application green. For instance there are methods of calculation of carbon emissions that are purely cost based, using those methods results in a value of C that will not not change if you make the application carbon aware or energy efficient.
+#### Operational Emissions
+
+To calculate the operational emissions associate with software, multiply the electricity consumption of the hardware the software is running on by the regional, granular marginal emissions rate. Because this standard uses a consequential approach, marginal emissions rates should be used for electricity consumption. The marginal emissions rate reflects the change in emissions assoicated with a change in demand. 
+
+Carbon aware software will optimize the timing and location of operation to minimize emissions associated with operation. This could consist of moving computation to regions with cleaner grid emissions or delaying jobs to cleaner periods (or a combination of both). Energy efficient software will also run on hardware that requires less energy to operate. All combined, these effects should be reflected in the total operational emissions. 
```

Comment:
Might be good also to include the discussion on re-architecting the solution after the sentence about running the solution on a different hardware.